### ISSUE TITLE
Show columns working incorrectly with multiple thead rows

### ISSUE THREAD

**Issue by nicolaas19** [wenzhixin/bootstrap-table#2173]
See JSFiddle: [https://jsfiddle.net/Lc2efe6b/](https://jsfiddle.net/Lc2efe6b/)

When you uncheck one (or more) of the columns, the column data is deleted from the tbody and the bottom most row of the thead, but not from the other thead rows.

**Comment by dabros** [comment]
prob related to #2006 or similar issues like #2051, and others i cant recall name of but a search should help you find (remove 'is:closed' and 'is:issue' filters)

Ive got massive work increase on my side, so i started fixing a few issues related to columns and rows being toggled off/on, as small bugs with including with colspan options, fixed-columns extension and similar

Cant finish these atm, but they wouldnt be hard to follow through as i share my workings and do make PR's for some already, though those (last i recall) need verification as there is more than one quirk to deal with.

**Comment by wenzhixin** [comment]
The same problem with #2006, close this one.
